presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -159,3 +159,25 @@ Async Data Cache and Prefetching
 
     Size of the SSD cache when async data cache is enabled.  Must be zero if
     ``async-data-cache-enabled`` is ``false``.
+
+``enable-old-task-cleanup``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``bool``
+    * **Default value:** ``true``
+
+    Enable periodic clean up of old tasks. This is ``true`` for Prestissimo,
+    however for presto-on-spark this defaults to ``false`` as zombie/stuck tasks
```

Comment:
```suggestion
    however for Presto on Spark this defaults to ``false`` as zombie/stuck tasks
```

Change suggested for consistency with existing documentation, see [Executing Presto on Spark](https://prestodb.io/docs/current/installation/spark.html).